include/mbedtls/ssl_internal.h

```diff
@@ -256,6 +256,18 @@
       + ( MBEDTLS_SSL_CID_OUT_LEN_MAX ) )
 #endif
 
+static uint32_t mbedtls_ssl_get_output_buflen( const mbedtls_ssl_context *ctx )
```

Comment:
Functions defined in headers should be `static inline`, not `static`. Otherwise you'll get a `-Wunused-function` warning whenever the header is included by a `.c` file that doesn't happen to use this particular function.